hack/e2e-suite/update.sh

```diff
@@ -1,144 +0,0 @@
-#!/bin/bash
```

Comment:
I know update.sh has been a bit flaky, but we typically leave the shell test in for a while after a rewrite to make sure that the port is at least as good as the shell test was. Can you put this file back with a TODO to remove it in a week or so after looking at jenkins history?